composer.json

```diff
@@ -16,9 +16,7 @@
         }
     ],
     "require": {
-        "sculpin/sculpin": "^2.1@dev",
-        "dflydev/embedded-composer": "^1.0@dev",
-        "kriswallsmith/assetic": "1.1.2",
+        "sculpin/sculpin": "dev-master as 3.0.0",
```

Comment:
we need to do this instead of ^3.0@dev until #414 is fixed

Reply:
#414 is fixed now, I think. Maybe this can be reworked?